gaztec/src/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,414 @@
+import { css } from "@emotion/react";
+import InputLabel from "@mui/material/InputLabel";
+import MenuItem from "@mui/material/MenuItem";
+import FormControl from "@mui/material/FormControl";
+import Select, { SelectChangeEvent } from "@mui/material/Select";
+import { AztecContext } from "../home/home";
+import { AztecEnv } from "../../config";
+import { createStore } from "@aztec/kv-store/indexeddb";
```

Comment:
Is this separate because e.g. a desktop app would not need to import this pkg?

Reply:
Yes, stores are plugins now!